spec/services/shopify_voider_spec.rb

```diff
@@ -6,14 +6,12 @@
   let(:transaction_id) { '0xDEADBEEF' }
   let(:order_id) { 'order_id' }
   let(:transaction_amount) { 1 }
-  let(:transaction) { double(:transaction, amount: transaction_amount, id: transaction_id) }
+  let(:transaction_instance) { double('transaction', amount: transaction_amount, id: transaction_id) }
+  let(:transaction_interface) { double('transaction_interface', find: transaction_instance) }
   let(:refunder_instance) { double('refunder_instance', perform: true) }
+  let(:refunder_class) { double('refunder_class', new: refunder_instance) }
 
-  before do
```

Comment:
I think this diff is a great example of how dependency injection can clean up tests.